### PR TITLE
Updated module w/ instructions on how to install locally (mac)

### PR DESCRIPTION
**Pull request by bentooth**
File structure and minor code updated to adhere to a [symfony](https://symfony.com/) bundle structure.

Updated the read me with the steps I took to get xcart 5.5 running locally on my mac

### CODE REVIEW

File: modules/ShipStation/Api/src/Controller/Customer/Api.php [hunk 1]
```diff
@@ -250,7 +250,6 @@ protected function getBillingInfo($objOrder = '')
                     $this->addFieldToXML("Country", $objBillingAddress->getCountry()->getCountry());
                     $this->addFieldToXML("CountryCode", $objBillingAddress->getCountry()->getCode());
                 }
-                $this->addFieldToXML("Company", $objBillingAddress->getCompanyName());
```
Comment (ricky) [inline modules/ShipStation/Api/src/Controller/Customer/Api.php]: Why was this removed? (Also on L289)

File: classes/XLite/Module/ShipStation/Api/changelog/5.4/1/2.log [hunk 2]
```diff
@@ -1,2 +0,0 @@
-Version 5.4.1.2
-02/10/2021 - [Changes] Import "company name" field with Orders
```
Comment (ricky) [inline classes/XLite/Module/ShipStation/Api/changelog/5.4/1/2.log]: Instead of deleting these, we move them to `modules/ShipStation/Api/docs/version_info`.
https://developer.x-cart.com/migration_guides/module_changelogs#where-to-put-changelogs

**Review by ricky — COMMENTED** [review]
It looks like this reverts https://github.com/shipstation/plugin-x-cart/pull/6. Was that intentional?

File: modules/ShipStation/Api/src/Controller/Customer/Api.php (hunk 1 above)
Comment (bentooth) [reply modules/ShipStation/Api/src/Controller/Customer/Api.php]: im not sure its from the support-5x branch. ill look into it

**Comment by bentooth** [comment]
> It looks like this reverts #6. Was that intentional?

its back!

File: modules/ShipStation/Api/docs/version_info/5.5/0.1/changelog.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Version 5.5.0.0
```
Comment (ricky) [inline modules/ShipStation/Api/docs/version_info/5.5/0.1/changelog.txt]: `Version: 5.5.0.1`? 

Also, why `5.5.0.0` and `5.5.0.1`? Is there a difference?

File: modules/ShipStation/Api/src/Controller/Customer/Api.php [hunk 4]
```diff
@@ -286,7 +286,7 @@ protected function getShippingInfo($objOrder = '')
                     $this->addFieldToXML("Country", $objShippingAddress->getCountry()->getCountry());
                     $this->addFieldToXML("CountryCode", $objShippingAddress->getCountry()->getCode());
                 }
-                $this->addFieldToXML("Company", $objShippingAddress->getCompanyName());
+                $this->addFieldToXML("Company", $objBillingAddress->getCompanyName());
```
Comment (ricky) [inline modules/ShipStation/Api/src/Controller/Customer/Api.php]: Looks like this should be `objShippingAddress`

File: modules/ShipStation/Api/src/ShipStationBundle.php [hunk 5]
```diff
@@ -54,7 +58,7 @@ public static function getMajorVersion()
      */
     public static function getMinorVersion() 
     {
-        return '1.2';
+        return '2';
```
Comment (ricky) [inline modules/ShipStation/Api/src/ShipStationBundle.php]: It looks like you can actually just remove `getMajorVersion` and `getMinorVersion` from this class. It's all in `main.yaml` now: https://developer.x-cart.com/5.4/misc/x-cart_versions#x-cart-54x-versioning

File: modules/ShipStation/Api/docs/version_info/5.5/0.1/changelog.txt (hunk 3 above)
Comment (bentooth) [reply modules/ShipStation/Api/docs/version_info/5.5/0.1/changelog.txt]: i already submitted 5.5.0.0 😅 so had to bump. you can't delete already submitted modules in xcart

File: modules/ShipStation/Api/src/Controller/Customer/Api.php (hunk 4 above)
Comment (bentooth) [reply modules/ShipStation/Api/src/Controller/Customer/Api.php]: yikes good catch

File: modules/ShipStation/Api/docs/version_info/5.5/0.1/changelog.txt (hunk 3 above)
Comment (ricky) [reply modules/ShipStation/Api/docs/version_info/5.5/0.1/changelog.txt]: This should still say `Version 5.5.0.1`

File: modules/ShipStation/Api/docs/version_info/5.5/0.2/changelog.txt [hunk 6]
```diff
@@ -0,0 +1,2 @@
+Version 5.5.0.0
```
Comment (ricky) [inline modules/ShipStation/Api/docs/version_info/5.5/0.2/changelog.txt]: `5.5.0.1`

**Review by ricky — APPROVED** [review]

**Review by ricky — CHANGES_REQUESTED** [review]
Can you `git rm` all of the `.DS_Store` files and add it to gitignore? Otherwise, people are gonna have trouble checking this out:

```shell
$ git checkout bvera/symfony-5.5-module
error: The following untracked working tree files would be overwritten by checkout:
	.DS_Store
Please move or remove them before you switch branches.
Aborting
```

Something like this should work:
```shell
$ find . -name .DS_Store -print0 | xargs -0 git rm -f --ignore-unmatch
$ echo '.DS_Store' >> .gitignore
```

**Comment by bentooth** [comment]
> Can you `git rm` all of the `.DS_Store` files and add it to gitignore? Otherwise, people are gonna have trouble checking this out:
> 
> ```shell
> $ git checkout bvera/symfony-5.5-module
> error: The following untracked working tree files would be overwritten by checkout:
> 	.DS_Store
> Please move or remove them before you switch branches.
> Aborting
> ```
> 
> Something like this should work:
> 
> ```shell
> $ find . -name .DS_Store -print0 | xargs -0 git rm -f --ignore-unmatch
> $ echo '.DS_Store' >> .gitignore
> ```

done!

**Review by ricky — APPROVED** [review]
